website_sale_cache/README.rst

```diff
@@ -0,0 +1,11 @@
+Website sale cache
+=================
+
+This module caches product public categories from module "website_sale". 
+That allows to significantly accelerate the loading of the page with a large number of product categories.
```

Comment:
The module allows to cache product website categories and significantly accelerate the page loading with a large number of product categories.